### ISSUE TITLE
Option to choose type of phone number

### ISSUE THREAD

**Issue by Moneybox76** [amoose136/call_or_sms_contact#5]
Hi,

it would be great if there is an option in the call workflow witch number (mobile or landline) will be called.

**Comment by amoose136** [comment]
Yes this has been something I wanted for a while but unfortunately implementing it is not so simple. In the end I would like to have things like contact pictures also show up in the search and then when you press enter it gives a list of number to call/sms from that contact if more than one exists. Right now on my system if more than one number for a contact exists then the contact just appears in the search multiple times but they all look the same and this is obviously not ideal.

**Comment by amoose136** [comment]
See also the last comment on #2 in the meantime.
